contact.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Sprint Challenge - Contact</title>
+
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto|Rubik"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/contact.css" />
+  </head>
+  <body>
+    <div class="contact-container-page">
+      <div class="contact-heading">
+        <img src="img/lambda-black.png" alt="logo" />
+        <header>
+          <nav>
+            <a href="index.html">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Products</a>
+            <a href="#">Blog</a>
+            <a href="contact.html">Contact</a>
+          </nav>
+        </header>
+      </div>
+      <div class="contact-form">
+        <form>
+          <section>
+            <label for="fname">First Name</label><br />
+            <input type="text" name="firstname" placeholder="First name.." />
+          </section>
+          <section>
+            <label for="lname">Last Name</label><br />
+            <input type="text" name="lastname" placeholder="Last name.." />
+          </section>
+          <section>
```

Comment:
it may be more clear for future developers reading the css if the class names were more descriptive like phone-number instead of phno